### ISSUE TITLE
Is the version hosted on cdnjs "official"?

### ISSUE THREAD

**Issue by jpasqua** [jsonform/jsonform#287]
I am using this library in a page served from an IoT device. I want all the libraries to be remote rather than stored on, and served by, the device. I see that there is a hosted version of jsonform on cdnjs:

https://cdnjs.cloudflare.com/ajax/libs/jsonform/2.2.0/jsonform.js

However, it does not appear to be the same as the version of the js that I downloaded from github. In particular, it contains a number of calls to `console.log` that aren't in the github version.

Is there a hosted version of jsonform.js that I can reliably reference from an embedded device?

**Comment by tchapi** [comment]
Hi @jpasqua 

The `jsonform.js` file is indeed the same version, the CDN JS version is automatically created on each release. Maybe you have been comparing the .min version that is no more in this repo (but still on CDN JS, I don't know why to be honest).

**Comment by jpasqua** [comment]
Thank you for your reply.I double checked and I think that you will find that if you diff these two files:

https://raw.githubusercontent.com/jsonform/jsonform/master/lib/jsonform.js
https://cdnjs.cloudflare.com/ajax/libs/jsonform/2.2.0/jsonform.js

You will get:

````
169a170
>     console.log('option Selected ' + $option.attr('value'))
171a173,175
>     // If the user explicitely chose one alternative, we must tell that this alternative has not a default
>     // value anymore
>     
1243a1248,1258
>     'onChange': function(evt, node) {
>       console.log('EVENT -------------------------', evt)
>       const value = evt.target.options.selectedIndex
>       _.each(node.children, function (child, idx) {
>         if (idx === value) {
>           console.log(child)
>           child.defaultValue = false
>           // child.value = "COUCOU"
>         }
>       });
>     },
1311a1327
>       console.log('onBeforeRender', children)
1313a1330
>         console.log('data.Value', activeChild)
1318a1336
>         console.log('non default', activeChild)
1319a1338
>           console.log(child.node, child.node.hasNonDefaultValue())
1323a1343
>         console.log('first one', activeChild)
1947c1967
< 
---
>   console.log('   comparing ' + this.value + ' to ' + this.defaultValue)
2060a2081
>   this.defaultValue = undefined
2180a2202
>     // console.log('compute', this)
2286a2309,2310
>     console.log('___________________________________________')
>     console.log('here first ' + this.value + ' ' + this.defaultValue)
2287a2312
>       console.log('   here second ' + this.value + ' ' + this.defaultValue)
2293a2319,2327
>       console.log('   here third ' + this.value + ' ' + this.defaultValue)
>       console.log(this)
>       // Special case for selects in a selectfieldset in an array (drag/dropping an item):
>       // The default values are automatically selected in the case of a select
>       // field, and thus, act as if they were selected by the user. Prevent that.
>       if (typeof this.value !== 'undefined' && (typeof this.defaultValue === 'undefined' || this.defaultValue === false) && 'enum' in this.schemaElement) {
>         //this.defaultValue = this.value === this.schemaElement['enum'][0]
>         console.log('IT IS THE DEFAULT: ' + this.defaultValue)
>       }
2295a2330,2331
> 
>       //console.log('   here third ' + this.value + ' ' + this.defaultValue)
2324a2361
>     console.log('FINALLY ' + this.value + ' ' + this.defaultValue + ' - '  + JSON.stringify(this.schemaElement))
2562c2599
<   // console.log("Form value",values);
---
>   console.log("Form value",values);
````

Sorry if I am missing something obvious.

**Comment by tchapi** [comment]
Hi @jpasqua 

You are right - it was due to the publishing of a "dev" version on NPM (not on Github). I fixed that in the 2.2.1 version which I just released. It should be ok now.

**Comment by jpasqua** [comment]
Thank you.
